simulations/vip-354/bscmainnet.ts

```diff
@@ -0,0 +1,165 @@
+      );
+      expect(await vUSDTStablecoins.interestRateModel()).to.equal(
+        RATE_MODELS.JumpRateModelV2_base0bps_slope1000bps_jump25000bps_kink8000bps,
+      );
+      for (const vToken of [vUSDTGameFi, vUSDTDeFi]) {
+        expect(await vToken.interestRateModel()).to.equal(
+          RATE_MODELS.JumpRateModelV2_base200bps_slope1500bps_jump25000bps_kink8000bps,
+        );
+      }
+    });
+
+    describe("old interest rate model parameters", async () => {
+      checkInterestRate(
+        RATE_MODELS.JumpRateModel_base0bps_slope875bps_jump25000bps_kink8000bps,
+        "vUSDT, vUSDC, vDAI (Core pool)",
+        {
+          base: "0",
+          multiplier: "0.0875",
+          jump: "2.5",
+          kink: "0.8",
+        },
+      );
+
+      checkInterestRate(RATE_MODELS.JumpRateModel_base0bps_slope875bps_jump50000bps_kink8000bps, "vFDUSD (Core pool)", {
+        base: "0",
+        multiplier: "0.0875",
+        jump: "5",
+        kink: "0.8",
+      });
+
+      checkInterestRate(
+        RATE_MODELS.JumpRateModelV2_base0bps_slope1000bps_jump25000bps_kink8000bps,
+        "vUSDT (Stablecoins pool)",
+        {
+          base: "0",
+          multiplier: "0.1",
+          jump: "2.5",
+          kink: "0.8",
+        },
+      );
+
+      checkInterestRate(
+        RATE_MODELS.JumpRateModelV2_base200bps_slope1500bps_jump25000bps_kink8000bps,
+        "vUSDT (GameFi, DeFi pools)",
+        {
+          base: "0.02",
+          multiplier: "0.15",
+          jump: "2.5",
+          kink: "0.8",
+        },
+      );
+    });
+  });
+
+  testVip("VIP-354", await vip354(), {
+    callbackAfterExecution: async txResponse => {
+      //await expectEvents(txResponse, [VTOKEN_CORE_POOL_ABI], ["NewMarketInterestRateModel"], [2]);
+      //await expectEvents(txResponse, [VTOKEN_IL_ABI], ["NewMarketInterestRateModel"], [2]);
+    },
+  });
+
+  describe("Post-VIP behavior", async () => {
+    it("has the new interest rate model addresses", async () => {
+      for (const vToken of [vUSDTCore, vUSDCCore, vDAICore]) {
+        expect(await vToken.interestRateModel()).to.equal(
+          RATE_MODELS.JumpRateModel_base0bps_slope1000bps_jump25000bps_kink8000bps,
+        );
+      }
+      expect(await vFDUSDCore.interestRateModel()).to.equal(
+        RATE_MODELS.JumpRateModel_base0bps_slope1000bps_jump50000bps_kink8000bps,
+      );
+      expect(await vUSDTStablecoins.interestRateModel()).to.equal(
+        RATE_MODELS.JumpRateModelV2_base0bps_slope375bps_jump25000bps_kink8000bps,
+      );
+      for (const vToken of [vUSDTGameFi, vUSDTDeFi]) {
+        expect(await vToken.interestRateModel()).to.equal(
+          RATE_MODELS.JumpRateModelV2_base200bps_slope1350bps_jump25000bps_kink8000bps,
+        );
+      }
+    });
+
+    describe("new interest rate model parameters", async () => {
+      checkInterestRate(
+        RATE_MODELS.JumpRateModel_base0bps_slope875bps_jump25000bps_kink8000bps,
```

Comment:
This is not the new IR for the markets vUSDT, vUSDC, vDAI
```suggestion
        RATE_MODELS.JumpRateModel_base0bps_slope1000bps_jump25000bps_kink8000bps,
```

... or we could get the address from one of the markets

Reply:
4f9d8250372be24de5b0f3194bacf4d9396fcb4f